test/e2e/test_helm_install_upgrade.go

```diff
@@ -63,11 +63,9 @@ func TestHelmUpgrade(t *testing.T) {
 		assertionsAfterUpgrade []assertion
 	}{
 		{
-			name: "upgrade from one before latest to latest minor",
-			// TODO: use renovate to bump the version in these 2 lines.
-			// https://github.com/Kong/gateway-operator/issues/121
-			fromVersion: "1.3.0",
-			toVersion:   "1.4.0",
+			name:        "upgrade from one before latest to latest minor",
+			fromVersion: "1.3.0", // renovate: datasource=docker packageName=kong/gateway-operator-oss depName=kong/gateway-operator-oss@only-patch
```

Comment:
This will require us to bump this manually on each minor/major release right? 

That's OK (as I believe solving this would be rather hard 🤔) Perhaps we could call this out in the release procedure https://github.com/Kong/gateway-operator/blob/c2c3235136c1266609275f895d2c30e17e535e70/.github/ISSUE_TEMPLATE/---release.md#L44? 

That point there deserves a change anyway given that this is being automated right?

Reply:
Yeah, that's right. I don't think it can be solved with just Renovate. I updated the release issue template as you requested: 532aee5